### DISCUSSION TITLE
Getting error to compile

### DISCUSSION THREAD

**Discussion by heheboii11** [Cyfrin/foundry-full-course-cu#3099] [Q&A] [answered]
Hey everyone I have recently working on some audit contests and I have cloned their git repo. And i have difficulties in compling the code. 
```
r-MacBook-Air 2024-11-autonomint-heheboii11 % forge build
Nothing to compile
```

This is the git repo link
https://github.com/sherlock-audit/2024-11-autonomint-heheboii11
I believe if the ```src``` file is not present in the project it will find difficulty in compling but i am not sure about how to solve this issue, may be some changes in ``foundry.toml``` or i don't know exactly, please if anyone know how to solve help me with this

**Comment by heheboii11** [comment, marked as answer]
found this , It did helped```https://github.com/Cyfrin/foundry-full-course-cu/discussions/481```

**Reply by EngrPips** [reply]
Thanks for the heads-up @heheboii11
